src/integrationTest/java/uk/gov/hmcts/darts/event/service/impl/DarStartHandlerTest.java

```diff
@@ -53,7 +53,7 @@ void throwsOnUnknownCourthouse() {
         event.setCaseNumbers(List.of("123"));
         event.setDateTime(today);
         assertThatThrownBy(() -> eventDispatcher.receive(event))
-            .isInstanceOf(DartsApiException.class);
+                .isInstanceOf(DartsApiException.class);
```

Comment:
formatting settings were temporarily wrong, and meant continuation indents where 8 instead of 4, if you merge from master, and reformat, it should fix this.

Reply:
did you make any real changes in this file?

Reply:
I think someone else made a similar change so after the merge it looks like I didn't make any changes.